src/cli.rs

```diff
@@ -83,6 +83,14 @@ Defaults to \"format\" if the NO_COLOR env is set and to \"none\" if stdout is n
     )]
     pub pretty: Option<Pretty>,
 
+    // Set formatting options, such as the amount of spaces per JSON indentation
+    //
+    // Different options need to be separated by a comma
+    // This only works when pretty is set to all or format
+    // Example: --format-options=json.indent:2
```

Comment:
```suggestion
    /// Set output formatting options. The only currently implemented option
    /// is json.indent.
    ///
    /// Different options need to be separated by a comma.
    ///
    /// Example: --format-options=json.indent:2
```

This has to be `///`, not `//`, or it won't show up in the help text. And it looks like HTTPie does have an option that applies regardless of `--pretty` (`headers.sort`), so maybe we shouldn't mention that.